src/app/clusters/boolean-state-server/CodegenIntegration.cpp

```diff
@@ -0,0 +1,90 @@
+/*
+ *
+ *    Copyright (c) 2025 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include "CodegenIntegration.h"
+#include <data-model-providers/codegen/CodegenDataModelProvider.h>
+
+using namespace chip;
+using namespace chip::app;
+using namespace chip::app::Clusters;
+using namespace chip::app::Clusters::BooleanState;
+using namespace chip::app::Clusters::BooleanState::Attributes;
+
+namespace {
+
+LazyRegisteredServerCluster<BooleanStateCluster> gServer;
```

Comment:
boolean state is a `Contact Sensor` for example and you may have multiple of such sensors. This means that a sinle server is insufficient and we should support multiple (including dynamic). We cannot say "if this exists it is always on endpoint 1"